src/ui/screens/SendFundScreen.tsx

```diff
@@ -57,17 +70,40 @@ export function SendFundScreen({navigation, route}: Props) {
           </View>
         ) : (
           <View style={styles.container}>
-            <Headline>Send</Headline>
+            <Headline>Send funds</Headline>
             <Padder scale={1} />
+            <InputLabel label="Enter amount" helperText="Type the amount you want to transfer" />
             <BalanceInput account={accountInfo} onChangeBalance={setAmount} />
             <Padder scale={1} />
+            <InputLabel
+              label="Send to address"
+              helperText="Scan a contact address or paste the address you want to send funds to."
+            />
             <TextInput
               autoComplete="off"
               placeholder="To"
-              value={to}
-              onChangeText={(nextValue) => setTo(nextValue)}
+              value={toAddress}
+              onChangeText={(nextValue) => setToAddress(nextValue)}
               right={<TextInput.Icon name="qrcode" onPress={() => setScanning(true)} />}
             />
+            {!isToAddressValid ? <HelperText type={'error'}>Enter a valid address</HelperText> : null}
```

Comment:
NIT 💅 
```suggestion
            {!isToAddressValid ? <HelperText type="error">Enter a valid address</HelperText> : null}
```

And for other cases also.